feature/experimental/isis/otg_tests/internal/session/session.go

```diff
@@ -46,9 +47,11 @@ const (
 	DUTAreaAddress = "49.0001"
 	ATEAreaAddress = "49.0002"
 	DUTSysID       = "1920.0000.2001"
-	ISISName       = "DEFAULT"
-	pLen4          = 30
-	pLen6          = 126
+	ATESysID       = "640000000001"
+	// TODO: Change the name to DEFAULT
+	ISISName = "osisis"
```

Comment:
What's the reason for the change?

Reply:
There is a problem with a vendor when the name is "DEFAULT". We're tracking the progress here https://github.com/openconfig/ka-fntest/issues/5